src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,26 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
 
+    const divBy3=(number%3)===0;
+    const divBy5=(number%5)===0;
+
+    if
+    (divBy3&&divBy5){
+        return 'FizzBuzz'
+}
+
+        if (divBy3){
+            return 'Fizz'
+    }
+    if
+        (divBy5){
+            return 'Buzz'
+    }
+  
+    
+        return number;
+    }
+        
+
+
+   
```

Comment:
The logic here is good, but you need to change the formatting somewhat. You never need more than one clear line in your code so you should remove all that at the end, if statements should have the condition on the same line as the `if` keyword, and the spacing should be uniform:

```
    const divBy3 = number % 3 === 0;
    const divBy5 = number % 5 === 0;

    if (divBy3 && divBy5) {
        return "FizzBuzz";
    }
    if (divBy3) {
        return "Fizz";
    }
    if (divBy5) {
        return "Buzz";
    }

    return number;
};
```